.github/workflows/generator_generic_slsa3.yml

```diff
@@ -129,6 +133,8 @@ jobs:
         id: detect
         continue-on-error: true
         uses: slsa-framework/slsa-github-generator/.github/actions/detect-workflow-js@main
+        with:
```

Comment:
same comment about the token being available

Reply:
it will be a different personal access token